docs/campaign/quests/substrate-pallets.md

```diff
@@ -17,32 +17,159 @@ If you are unsure which chain state function to call, you can also use this step
+
+AirLyft simplifies the process of creating Substrate Pallet tasks, making it as straightforward as using Polkadotjs. Let's dive deep into it.
+
+#### Check Substrate Balance
+
+- This option allows you to verify whether a user has a minimum required balance of a specific token in their Substrate-based wallet. This is particularly useful for campaigns that reward users for holding or maintaining a certain balance of an asset, ensuring they have an active presence in the ecosystem before participating in a quest.
+
+1. Select the Substrate Balance option, as shown in the screenshot below.
+
+   ![](../../images/palletsOptions.png)
+
+2. Enter basic information such as the **Quest Name, a brief Description, the cFuel allocation, and XP rewards**. This information helps define the purpose of the quest and the incentives for completing it.
+
+   ![](../../images/palletAssetHubAssetBalance.png)
+
+3. To configure the **Task Condition & Recurrence**, refer to the [Task Condition & Recurrence](../task-condition-and-recurrence.md) page.
+
+4. Choose the **Blockchain** where the balance should be checked (e.g., Polkadot AssetHub, Subsocial, etc.). Enter the **Minimum balance** that users must hold to be eligible for completing the quest. This ensures that only users meeting the criteria can proceed.
+
+5. Once all details are filled in, click the **Add Task** button to complete the process. The Substrate Balance quest is now set up and ready for users to participate.
+
+#### Check Substrate Stake Amount
+
+- Staking plays a vital role in securing blockchain networks, and this option enables you to check if a user has staked a specific amount of tokens. By using this task, you can create campaigns that incentivize users to participate in staking, ensuring they are actively contributing to network security before earning rewards.
+
+1. Navigate to the "Quests" tab in your campaign page and select the Substrate Pallets quest. From the available options, choose **Substrate Stake Amount**, as shown in the image below.
+
+   ![](../../images/palletsOptions.png)
+
+2. Fill in the basic quest information (name, description, cFuel, and XP).
+
+   ![](../../images/palletStake.png)
+
+3. To configure the **Task Condition & Recurrence**, refer to the [Task Condition & Recurrence](../task-condition-and-recurrence.md) page.
+
+4. Select the **Blockhain** where the staking amount should be checked and enter the **Minimum amount** that users must have to qualify.
+
+5. Then click on **Add Task** button. That's it your Substrate Pallets quest has been created.
+
+#### Check Substrate AssetHub Asset Balance
+
+- Many blockchain ecosystems have multiple assets, and this option lets you verify if a user holds a required amount of a specific token on Substrate AssetHub. This feature is ideal for projects looking to engage users who hold specific assets, ensuring they meet the criteria before progressing in a campaign.
```

Comment:
Not needed `Many blockchain ecosystems have multiple assets,`

Reply:
Not needed `This feature is ideal for projects looking to engage users who hold specific assets, ensuring they meet the criteria before progressing in a campaign.`